### PR TITLE
Adding UX building blocks updates to the 2.4.0 Release Note

### LINKED ISSUE #7803: MRTK 2.4 Release Notes - UX Building blocks update
## Describe the bug
Add following items to 2.4.0 release note

- Button Config Helper added to Pressable Buttons
- Hand Menu Improvements
- Hand Menu Examples update
- Dialog (Experimental)
- Dock (Experimental)
-

### PR DESCRIPTION
**Pull request by cre8ivepark**
Adding UX building blocks updates to the 2.4.0 Release Note

Fixes #7803 

## Added items
**Button Config Helper added to Pressable Buttons**
**Toggle Checkbox/Switch/Radio Buttons**
**Hand Menu Improvements**
**Hand Menu Examples update**
**Dialog (Experimental)**
**Dock (Experimental)**

## Page
https://github.com/microsoft/MixedRealityToolkit-Unity/blob/cre8ivepark-patch-1/Documentation/ReleaseNotes.md

2020-05-05 20_27_12-Editing MixedRealityToolkit-Unity_ReleaseNotes md at prerelease_2 4 0_stabilizat
2020-05-05 20_32_35-Editing MixedRealityToolkit-Unity_ReleaseNotes md at prerelease_2 4 0_stabilizat

### CODE REVIEW

**Comment by davidkline-ms** [comment]
@cre8ivepark, there is a doc generation error that needs to be fixed.

An non-relative doc link was found in D:\a\1\s\Documentation\ReleaseNotes.md at line 58 

Thanks!

File: Documentation/ReleaseNotes.md [hunk 1]
```diff
@@ -54,6 +54,38 @@ This version of MRTK modifies the layout of the MRTK folder structure. This chan
 
 The [MRTK Toolbox](README_Toolbox.md) is a Unity editor window utility that makes it easy to discover and spawn MRTK UX prefab components into the current scene. Items can be filtered in view by using the search bar at the top of the window. The toolbox window is designed to spawn MRTK out-of-box prefabs into the current scene.
 
+**Button Config Helper added to Pressable Buttons**
+![Button Config Helper](https://user-images.githubusercontent.com/9789716/70167111-e3175600-167a-11ea-9c52-444509c06105.gif)
+This new feature makes it easy to change the icon and text of the buttons. Icon supports quad, sprite, and TextMesh Pro's SDF font texture. See MRTK's [Button documentation](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/README_Button.html#how-to-change-the-icon-and-text) for the details.
```
Comment (keveleigh) [inline Documentation/ReleaseNotes.md:59]: Looks like doc validation failed on this. I think just referencing the .md file is good and gets updated during the docs build to point to the correct .html page.

```suggestion
This new feature makes it easy to change the icon and text of the buttons. Icon supports quad, sprite, and TextMesh Pro's SDF font texture. See MRTK's [Button documentation](README_Button.md#how-to-change-the-icon-and-text) for the details.
```

File: Documentation/ReleaseNotes.md [hunk 2]
```diff
@@ -54,6 +54,38 @@ This version of MRTK modifies the layout of the MRTK folder structure. This chan
...
+**Button Config Helper added to Pressable Buttons**
+![Button Config Helper](https://user-images.githubusercontent.com/9789716/70167111-e3175600-167a-11ea-9c52-444509c06105.gif)
+This new feature makes it easy to change the icon and text of the buttons. Icon supports quad, sprite, and TextMesh Pro's SDF font texture. See MRTK's [Button documentation](README_Button.md#how-to-change-the-icon-and-text) for the details.
+
+**New HoloLens 2-style Toggle Buttons - Checkbox, Switch, Radio**
+<br/><img src="https://user-images.githubusercontent.com/13754172/75299797-df631d80-57ea-11ea-8857-8ef647df0aca.gif" width="450">
+<br/><img src="https://user-images.githubusercontent.com/13754172/75299783-d6724c00-57ea-11ea-88b1-85e4a585212f.gif" width="450">
+
+
+**Hand Menu Improvements**
```
Comment (keveleigh) [inline Documentation/ReleaseNotes.md:66]: Nit: extra whitespace

```suggestion
<br/><img src="https://user-images.githubusercontent.com/13754172/75299783-d6724c00-57ea-11ea-88b1-85e4a585212f.gif" width="450">

**Hand Menu Improvements**
```

**Pull request merged**